pkg/common/cns-lib/volume/manager.go

```diff
@@ -951,3 +956,54 @@ func (m *defaultManager) ConfigureVolumeACLs(ctx context.Context, spec cnstypes.
 	}
 	return err
 }
+
+// RegisterDisk registers a virtual disk as a First Class Disk.
+// This method helps in registering VCP volumes as FCD using vslm endpoint
+func (m *defaultManager) RegisterDisk(ctx context.Context, path string, name string) (string, error) {
```

Comment:
Can you clarify `path` and `name` in the function comment?

Reply:
Done